packages/main/src/themes/Calendar.css

```diff
@@ -15,7 +15,7 @@
     width: var(--_ui5_calendar_width);
     padding: var(--_ui5_calendar_top_bottom_padding) var(--_ui5_calendar_left_right_padding) 0;
     display: flex;
-    flex-direction: column-reverse;
+    flex-direction: column;
     justify-content: flex-end;
```

Comment:
Seems to me that justify-content: flex-end is not needed anymore as it results into a misaligned content vertically in the month view.